app/config/config.yml

```diff
@@ -31,12 +31,12 @@ framework:
 
 doctrine:
     dbal:
-        driver: '%database_driver%'
         host: '%database_host%'
         port: '%database_port%'
         dbname: '%database_name%'
         user: '%database_user%'
         password: '%database_password%'
+        driver: pdo_mysql
```

Comment:
It's pointless to have database `driver` configurable via environment variable when the `server_version` is fixed here. It's also something that does not change in an application, so following [Symfony Best Practices](https://symfony.com/doc/3.4/best_practices/configuration.html#application-related-configuration):

> ### Application-Related Configuration
> > Define the application behavior related configuration options in the app/config/config.yml file.
>
> Defining these values in parameters.yml file would add an extra layer of configuration that's not needed because you don't need or want these configuration values to change on each server.